### DISCUSSION TITLE
Hide a div

### DISCUSSION THREAD

**Discussion by abraolira** [cdnjs/cdnjs#14244] [General]
How can I hide a div to prevent it from appearing in the PDF file? I have created a rule in CSS, but apparently it doesn't work with this pdf.

**Comment by MattIPv4** [comment]
Hi there, I don't think this question relates to the functionality of cdnjs as a CDN. If you're using a library hosted on cdnjs and are looking for help/support with that, you will need to reach out to that library's specific community.
